examples/3d/parenting.rs

```diff
@@ -52,6 +61,22 @@ fn setup(
                 ..default()
             });
         });
+    // parent "empty"
+    commands
+        .spawn_bundle((
+            Transform::from_xyz(2.0, 0.0, 1.0),
+            GlobalTransform::default(),
+        ))
```

Comment:
There is a `TransformBundle` that was introduced in #3054 for that case 🙂

Reply:
Nice, yes my IDE did showed it to me but couldn't go to its source and I couldn't find the docs on the internet either ^^ I'm going to adapt my code with it, might as well advertise that this bundle exists too :)